src/Razor/src/Microsoft.AspNetCore.Razor.Language/Syntax/Syntax.xml

```diff
@@ -86,6 +86,10 @@
     <Field Name="Value" Type="RazorBlockSyntax" Optional="true" />
     <Field Name="ValueSuffix" Type="MarkupTextLiteralSyntax" Optional="true" />
   </Node>
+  <Node Name="MarkupMiscAttributeContentSyntax" Base="MarkupSyntaxNode">
```

Comment:
I added this node to hold all content in the tag that are not valid attributes. So ultimately `MarkupTagBlockSyntax` will contain,
Openangle,
OptionalMetacode (`!`),
Name,
ListOfAttributes (This can be either `MarkupAttributeBlockSyntax`, `MarkupMinimizedAttributeBlockSyntax` or `MarkupMiscAttributeContentSyntax`)
OptionalForwardSlash
CloseAngle